### DISCUSSION TITLE
is there any plan to add EdgeToolbar?

### DISCUSSION THREAD

**Discussion by zidni-bwi** [bcakmakoglu/vue-flow#905] [Ideas]
NodeToolbar is very helpful for node customization I use it quite a lot but for edge still use manual way I think it would be great if edge also got a feature like NodeToolbar

**Comment by bcakmakoglu** [comment]
Have you considered using the `EdgeLabelRenderer`?

[Check this example](https://vueflow.dev/examples/edges/) and open up the `CustomEdge.vue` file to see how it's used.

**Comment by heaven-born** [comment]
I would also request a version of  `EdgeLabelRenderer`  for rendering labels/bars near the connection points. The use case is when I want to do a transition from node to node. In this case, I would like to click on this kind of label in order to start transitioning to the opposite direction of the edge.
